packages/yew-router/src/components/link.rs

```diff
@@ -24,6 +24,9 @@ where
     pub query: Option<Q>,
     #[prop_or_default]
     pub disabled: bool,
+    /// [`NodeRef`](yew::html::NodeRef) for the `<a>` element.
+    #[prop_or_default]
+    pub a_ref: NodeRef,
```

Comment:
```suggestion
    pub anchor_ref: NodeRef,
```

I think `a_ref` should be given a more descriptive name.

Reply:
Yeah I agree, I'll change right now.